pkgs/development/tools/misc/cookiecutter/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, fetchurl, pythonPackages }:
+
+pythonPackages.buildPythonPackage rec {
+  version = "1.3.0";
+  name = "cookiecutter-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/audreyr/cookiecutter/archive/${version}.tar.gz";
+    md5 = "378670fe456957eb3c27ddaef60b2b24";
+  };
```

Comment:
could you use sha256 here? 

Also, in my opinion, this should be moved to `python-packages.nix`.
You can still put an entry in `all-packages.nix`, e.g. `cookiecutter = pythonPackages.cookiecutter;`